tests/unittests/test_error_handling.py

```diff
@@ -1,59 +1,47 @@
 import unittest
+import json
 from unittest import mock
 import requests
 from tap_mixpanel import LOGGER, client
 
-# mock responce
-
-
-class Mockresponse:
-    def __init__(self, resp, status_code, content=[""], headers=None, raise_error=False):
-        self.json_data = resp
-        self.status_code = status_code
-        self.content = content
-        self.headers = headers
-        self.raise_error = raise_error
-        self.text = {}
-        self.reason = "error"
-
-    def prepare(self):
-        return (self.json_data, self.status_code, self.content, self.headers, self.raise_error)
-
-    def raise_for_status(self):
-        if not self.raise_error:
-            return self.status_code
-
-        raise requests.HTTPError("mock sample message")
-
-    def json(self):
-        return self.text
-
+# Mock response
+def get_mock_http_response(content, status_code):
+    contents = content
```

Comment:
@savan-chovatiya No need of this variable. You can directly use 'content' variable at line 13

Reply:
Removed `contents` variable and directly used `content`.